fixtures/pages-functions-with-routes-app/tests/index.test.ts

```diff
@@ -12,6 +12,7 @@ describe("Pages Functions with custom _routes.json", () => {
 			"public",
 			["--port=0", "--inspector-port=0"]
 		));
+		console.log(ip, port);
```

Comment:
Is this intentional?

Reply:
https://github.com/cloudflare/workers-sdk/pull/7069/commits/93e02541648acb60957b3c93f42b83aeeef2a890